src/components/btn-cell-renderer.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
Please change name of component to PascalCase.

Reply:
Changed